============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-8.3.3, pluggy-1.5.0
rootdir: /home/james/projects/cs6223/our_proptest_data
plugins: hypothesis-6.112.2, timeout-2.3.1
collected 25 items

doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py FFFFF    [ 20%]
doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py FFFFF    [ 40%]
doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py FFFFF    [ 60%]
doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py FFFFF    [ 80%]
doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py F.FFF    [100%]

=================================== FAILURES ===================================
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py:9: assert (False)
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py:15: AssertionError: assert (True and False)
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py:21: assert (False)
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py:27: assert (True and False)
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py:33: assert (False)
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py:9: assert -1 > 0
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py:15: assert 0 > 0
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py:21: assert -1 > 0
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py:27: assert -2 > 0
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py:33: assert -100 > 0
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py:11: assert 2 == 1
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py:18: assert 0 == 1
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py:25: assert 0 == 1
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py:33: assert 2 == 1
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py:42: assert 5 == 1
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py:8: AssertionError: assert (Decimal('0') / Decimal('1')) != Decimal('0')
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py:13: AssertionError: assert (Decimal('1') / Decimal('1')) == Decimal('0')
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py:18: AssertionError: assert (Decimal('1') / Decimal('2')) == Decimal('1')
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py:23: AssertionError: assert (Decimal('2') / Decimal('1')) == Decimal('1')
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py:28: AssertionError: assert (Decimal('1') / Decimal('2')) == Decimal('1')
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py:7: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_decimal_Decimal_as_integer_ratio_1 returned (0, 1) instead.
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py:37: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_decimal_Decimal_as_integer_ratio_3 returned (42, 1) instead.
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py:50: hypothesis.errors.FailedHealthCheck: Tests run under @given should return None, but test_violation_of_decimal_Decimal_as_integer_ratio_4 returned (1, 1) instead.
/home/james/projects/cs6223/our_proptest_data/doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py:64: exceptiongroup.ExceptionGroup: Hypothesis found 2 distinct failures. (2 sub-exceptions)
=========================== short test summary info ============================
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py::test_violation_of_decimal_Decimal_as_integer_ratio_1
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py::test_violation_of_decimal_Decimal_as_integer_ratio_2
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py::test_violation_of_decimal_Decimal_as_integer_ratio_3
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py::test_violation_of_decimal_Decimal_as_integer_ratio_4
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_1.py::test_violation_of_decimal_Decimal_as_integer_ratio_5
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py::test_violation_of_decimal_Decimal_as_integer_ratio_1
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py::test_violation_of_decimal_Decimal_as_integer_ratio_2
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py::test_violation_of_decimal_Decimal_as_integer_ratio_3
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py::test_violation_of_decimal_Decimal_as_integer_ratio_4
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_2.py::test_violation_of_decimal_Decimal_as_integer_ratio_5
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py::test_violation_of_decimal_Decimal_as_integer_ratio_1
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py::test_violation_of_decimal_Decimal_as_integer_ratio_2
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py::test_violation_of_decimal_Decimal_as_integer_ratio_3
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py::test_violation_of_decimal_Decimal_as_integer_ratio_4
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_3.py::test_violation_of_decimal_Decimal_as_integer_ratio_5
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py::test_violation_of_decimal_Decimal_as_integer_ratio_1
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py::test_violation_of_decimal_Decimal_as_integer_ratio_2
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py::test_violation_of_decimal_Decimal_as_integer_ratio_3
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py::test_violation_of_decimal_Decimal_as_integer_ratio_4
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_4.py::test_violation_of_decimal_Decimal_as_integer_ratio_5
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py::test_violation_of_decimal_Decimal_as_integer_ratio_1
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py::test_violation_of_decimal_Decimal_as_integer_ratio_3
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py::test_violation_of_decimal_Decimal_as_integer_ratio_4
FAILED doc_only/mutants_2/decimal.Decimal.as_integer_ratio/mutant_5.py::test_violation_of_decimal_Decimal_as_integer_ratio_5
========================= 24 failed, 1 passed in 9.98s =========================
